client/src/templates/Challenges/video/Show.js

```diff
@@ -178,20 +183,26 @@ export class Project extends Component {
                 <ChallengeTitle isCompleted={isChallengeCompleted}>
                   {blockNameTitle}
                 </ChallengeTitle>
-                <Spacer />
-                <div className='text-center'>
-                  <LoadablePlyr
-                    sources={{
-                      type: 'video',
-                      sources: [
-                        {
-                          src: videoId,
-                          provider: 'youtube'
-                        }
-                      ]
+                <div className='video-wrapper'>
+                  {!this.state.videoIsLoaded ? (
+                    <div className='video-placeholder-loader'>
+                      <Loader />
+                    </div>
+                  ) : null}
+                  <YouTube
+                    className={
+                      this.state.videoIsLoaded
+                        ? 'display-youtube-video'
+                        : 'hide-youtube-video'
+                    }
+                    onReady={this.videoIsReady}
+                    opts={{
+                      rel: 0,
+                      width: '960px',
+                      height: '540px'
```

Comment:
```suggestion                   
                    opts={{
                      playerVars: {
                        rel: 0
                      },
                      width: '960px',
                      height: '540px'
```

Reply:
As discussed, we would put this in a seprate commit for sake of clean history.